jsonnet/telemeter/client/kubernetes.libsonnet

```diff
@@ -244,6 +244,25 @@ local securePort = 8443;
                 record: 'cluster:telemetry_selected_series:count',
                 expr: 'max(federate_samples - federate_filtered_samples)',
               },
+              {
+                expr: |||
+                  sum by (namespace) (
```

Comment:
Why `by (namespace)`? The telemeter client is running in a single namespace, as mentioned in `annotation.description` below. Also, in github.com/openshift/telemeter/pkg/forwarder/forwarder.go I see no labels added, so the namespace should be added as a target label, which will always be `openshift-monitoring` because this is running just in that namespace

Reply:
as stated in https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/alerting-consistency.md, it's a good practice to have a namespace label attached to the alert as it enables better alert routing (e.g. all alerts from openshift-monitoring should go to the team responsible for the monitoring components). The namespace could be hard-coded in the fixed alert's labels or (preferred) come from the metric itself.